deploy/olm-catalog/community-kubevirt-hyperconverged/1.12.0/manifests/kubevirt-hyperconverged-operator.v1.12.0.clusterserviceversion.yaml

```diff
@@ -3267,6 +3267,7 @@ spec:
                   value: quay.io/kubevirt-ui/kubevirt-plugin@sha256:4ed1f37116cbaf883edd8783201d091fbbe121fcb84e7184ee531d3b1abd77ab
                 - name: KV_CONSOLE_PROXY_IMAGE
                   value: quay.io/kubevirt-ui/kubevirt-apiserver-proxy@sha256:38293a1050bce9c7cb786cbe8719be1edb91a922a6c583eedb61e56729ed4c72
+                - name: WASP_AGENT_IMAGE
```

Comment:
the value of the env var should have been set in the CSV

Reply:
@orenc1 Can you please help me with adding it? I am not sure whether I need to:
* Add it manually
* Run some make target
* Add some missing piece of code/script and then run some make target